rocAL/CMakeLists.txt

```diff
@@ -189,6 +193,12 @@ if(NOT HALF_FOUND)
     set(BUILD_ROCAL false)	
     message("-- ${Yellow}NOTE: rocAL library requires HALF, Not Found ${ColourReset}")	
 endif()
+if (${libsnd_LIBS} STREQUAL "libsnd_LIBS-NOTFOUND")
```

Comment:
`if( NOT SndFile_FOUND )` or `if(SNDFILE_FOUND)`

Try this and see if it works